Explorer/Assets/DCL/SDKComponents/TextShape/Renderer/Factory/PoolTextShapeRendererFactory.cs

```diff
@@ -0,0 +1,30 @@
+using DCL.Optimization.Pools;
+using DCL.SDKComponents.TextShape.Fonts;
+using UnityEngine;
+
+namespace DCL.SDKComponents.TextShape.Renderer.Factory
+{
+    /// <summary>
+    /// Is not thread safe
```

Comment:
We are using ThreadSafeComponentPool here, is it unsafe? Can this factory be used on multiple threads?

Reply:
I saw comments above ThreadSafeObjectPool<T> in DCL.Optimization.Pools.ComponentPool<T> that it's not thread-safe. So it's need to be clarified. I haven't tests now that covers multithreading use-cases

```
/// <summary>
///     It is not thread-safe
/// </summary>
private readonly ThreadSafeObjectPool<T> objectPool;
```